src/components/chat/chat-panel.tsx

```diff
@@ -26,18 +27,56 @@ export const ChatPanel: React.FC<IProps> = ({ user, activeNavTab, focusDocument,
   const tileComments = comments?.filter(comment => comment.tileId === focusTileId);
   const postedComments = focusTileId ? tileComments : documentComments;
   const postCommentMutation = usePostDocumentComment();
+  const ui = useUIStore();
+  const localDocuments = useLocalDocuments();
   const postComment = useCallback((comment: string) => {
+    if (focusDocument) {
+      const localDocument = localDocuments.getDocument(focusDocument);
+      const numComments = postedComments ? postedComments.length : 0;
+      if (localDocument) {
+        if (ui.selectedTileIds.length === 0) {
+          if (numComments < 1) {
+            Logger.logDocumentEvent(LogEventName.CHAT_PANEL_ADD_INITIAL_COMMENT_FOR_DOCUMENT, localDocument,
+              undefined, comment);
+          } else {
+            Logger.logDocumentEvent(LogEventName.CHAT_PANEL_ADD_RESPONSE_COMMENT_FOR_DOCUMENT, localDocument,
+              undefined, comment);
+          }
+        } else {
+          if (numComments < 1) {
+            Logger.logDocumentEvent(LogEventName.CHAT_PANEL_ADD_INITIAL_COMMENT_FOR_TILE, localDocument,
+              undefined, comment);
+          } else {
+            Logger.logDocumentEvent(LogEventName.CHAT_PANEL_ADD_RESPONSE_COMMENT_FOR_TILE, localDocument,
+              undefined, comment);
```

Comment:
Same as above about the ternary,  You could leave these as simple `Logger.log` for now and we will eventually replace them with `logTileEvent`, but that is probably not worth the time.